gtda/homology/cubical.py

```diff
@@ -26,6 +26,13 @@ class CubicalPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
     dimensions and at different scales is summarised in the corresponding
     persistence diagram.
 
+    **Important notes**:
+        - Persistence diagrams produced by this class must be interpreted with
+          care due to the presence of padding triples which carry no
+          information. See :meth:`transform` for additional information.
+        - In homology dimension 0, :meth:`transform` automatically removes one
```

Comment:
This is not automatic anymore but only if `reduced_homology`, right?

Reply:
This is valid for all homology transformer.

Reply:
Great, thanks for spotting this!

Reply:
Deleted from all homology transformers.